### ISSUE TITLE
Add Toggle for trailing punctuation inclusion

### ISSUE THREAD

**Issue by chrisgrieser** [chrisgrieser/obsidian-smarter-md-hotkeys#18]
Based on the idea by @Aaron-Bopp in #17, there should be a toggle for inclusion/exclusion of trailing punctuation.

**Comment by ReaderGuy42** [comment]
Just wanted to add my support for this :) Especially with the smarter highlights and bolds, etc. it would be handy to also have the smarter stuff include other punctuation. I noticed because I was trying to comment out something, but the smarter comment function wasn't including the asterisks, etc.
Thanks :)

**Comment by ScottCTD** [comment]
I agree. Smart Bold always makes the trailing colon bold which is really annoying.

For example, 
```markdown
a word:
```
If I selected the `word` and then Smart Bold it, then the trailing colon will also be bold. This doesn't respect my selection.
